frontend/src/static/css/style.scss

```diff
@@ -364,16 +364,9 @@ header {
 
     &--bar {
       background-color: mui-color('red');
-      height: .3rem;
-      margin-top: .1rem;
-    }
-
-    &--bar--fileType {
-      display: inline-block;
+      height: 4px;
+      margin-top: 2px;
       float: left;
-      height: .3rem;
-      margin-top: .1rem;
```

Comment:
the attribute `float` should come before height and margin

Reply:
Updated just now.